integration-tests/ccip-tests/testconfig/global.go

```diff
@@ -113,6 +113,23 @@ func NewConfig() (*Config, error) {
 		return nil, errors.Wrap(err, ErrReadConfig)
 	}
 
+	// read secrets for all products
```

Comment:
should this be part of this PR? not related to VRF

Reply:
@iljapavlovs it's related to the reusable workflow and `test_config_override_path` that is part of this PR.